packages/x-date-pickers/src/PickersTextField/PickersFilledInput/PickersFilledInput.tsx

```diff
@@ -1,31 +1,40 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { FormControlState, useFormControl } from '@mui/material/FormControl';
 import { styled, useThemeProps } from '@mui/material/styles';
 import { shouldForwardProp } from '@mui/system';
 import { refType } from '@mui/utils';
 import composeClasses from '@mui/utils/composeClasses';
 import {
   pickersFilledInputClasses,
   getPickersFilledInputUtilityClass,
+  PickersFilledInputClasses,
 } from './pickersFilledInputClasses';
 import { PickersInputBaseProps, PickersInputBase } from '../PickersInputBase';
 import {
   PickersInputBaseRoot,
   PickersInputBaseSectionsContainer,
 } from '../PickersInputBase/PickersInputBase';
+import { PickersTextFieldOwnerState } from '../PickersTextField.types';
+import { usePickerTextFieldOwnerState } from '../usePickerTextFieldOwnerState';
 
 export interface PickersFilledInputProps extends PickersInputBaseProps {
   disableUnderline?: boolean;
   hiddenLabel?: boolean;
 }
 
+export interface PickersFilledInputOwnerState extends PickersTextFieldOwnerState {
+  /**
+   * If `true`, the input  have an underline, `false` otherwise.
```

Comment:
```suggestion
   * If `true`, the input has an underline, if `false` it has not.
```

Reply:
I rephrased to be coherent with the other properties